shared/lib/constants.py

```diff
@@ -18,8 +18,8 @@
 import frozendict
 
 _RATE_WORDS_TO_SKIP = [
-    "birth", "change", "death", "exchange", "fertility", "literacy",
-    "mortality", "participation", "unemployment", "withdrawal"
+    "birth", "change", "exchange", "fertility", "literacy", "participation",
```

Comment:
This could use a useful comment.  Something like, or better than:

"These are words that come before `rate`, which together mean a specific metric (like "fertility rate").  So if "fertility rate" appears in the query, we will not infer PerCapita heuristic.  Note that we expect to find the full phrase with the word `rate` in the NL index description."

Reply:
done